pkg/client/mapiserver/volume_delete.go

```diff
@@ -14,31 +10,6 @@ const (
 
 // DeleteVolume will request maya-apiserver to delete volume (vname)
 func DeleteVolume(vname string, namespace string) error {
-
-	_, err := GetStatus()
-	if err != nil {
-		return util.MAPIADDRNotSet
-	}
-
-	url := GetURL() + "/latest/volumes/delete/" + vname
-	req, err := http.NewRequest("GET", url, nil)
-	if err != nil {
-		return err
-	}
-
-	req.Header.Set("namespace", namespace)
-	c := &http.Client{
-		Timeout: timeoutVolumeDelete,
-	}
-	resp, err := c.Do(req)
-	if err != nil {
-		return err
-	}
-
-	defer resp.Body.Close()
-	code := resp.StatusCode
-	if code != http.StatusOK {
-		return fmt.Errorf("Status error: %v ", http.StatusText(code))
-	}
-	return nil
+	_, err := getRequest(GetURL()+"/latest/volumes/delete/"+vname, namespace, false)
```

Comment:
can we define go constant for endpoints ? i.e. deleteEndpoint, createEndpoint ?

Reply:
yup we can

Reply:
but i think we should not do it in this pr